packages/editor/src/components/widgets/table/row/ReorderRow.css

```diff
@@ -5,12 +5,12 @@
 .dnd-row-handle {
   cursor: grab;
   width: 20px;
-}
-.dnd-row-handle i {
-  display: flex;
-  justify-content: flex-end;
-}
+  i {
+    display: flex;
+    justify-content: flex-end;
+  }
 
-.dnd-row-handle .ivy-change-type {
-  padding: var(--size-2);
+  .ivy-change-type {
+    padding: var(--size-2);
+  }
```

Comment:
Oh, I see. Sorry. I will adjust the CSS like this:
```
.dnd-row-handle .ivy-change-type {
  display: flex;
  justify-content: flex-end;
  padding: var(--size-2);
}
```
Because if I were to work with the <i> tag, the 'x' icon of the error message would also be affected, and I deliberately want to only affect the change-type icon with the padding.

Reply:
Yes of course this is the other solution :)